rules/eslint/best-practices/walmart.js

```diff
@@ -58,23 +62,25 @@ module.exports = {
     "no-lone-blocks": 2,
     // disallow creation of functions within loops
     "no-loop-func": 2,
+    // disallow the use of magic numbers
+    "no-magic-numbers": 0,
```

Comment:
I'm soooooooooo tired of calling people out on this in code reviews. Thank you ESLint!

![image](https://cloud.githubusercontent.com/assets/940667/11983487/ac8a9b66-a968-11e5-8688-4fd83718779d.png)